ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .catalog import Catalog
```

Comment:
I think you should include all the things you include from external libs here. And then actually include them from the package level instead of the underlying files.